.goreleaser.yml

```diff
@@ -22,92 +22,54 @@ checksum:
   # Default is `{{ .ProjectName }}_{{ .Version }}_checksums.txt`.
   name_template: "{{ .ProjectName }}_checksums.txt"
 
-nfpm:
-  name_template: "{{ .ProjectName }}_{{ .Os }}"
-
-  # Your app's vendor.
-  # Default is empty.
-  vendor: Ibotta
-
-  # Your app's homepage.
-  # Default is empty.
-  homepage: https://github.com/Ibotta/sopstool
-
-  # Your app's maintainer (probably you).
-  # Default is empty.
-  maintainer: Ibotta <oss@ibotta.com>
-
-  # Your app's description.
-  # Default is empty.
-  description: "A multi-file wrapper for Mozilla sops"
-
-  # Your app's license.
-  # Default is empty.
-  license: Apache 2.0
-
-  # Formats to be generated.
-  formats:
-    - deb
-    - rpm
-
-  # Packages your package depends on.
-  dependencies:
-    - sops
-
-brew:
-  # # Reporitory to push the tap to.
-  github:
-    owner: Ibotta
-    name: homebrew-public
-
-  # Git author used to commit to the repository.
-  # Defaults are shown.
-  commit_author:
-    name: sopstoolGoreleaser
-    email: oss@ibotta.com
-
-  # Folder inside the repository to put the formula.
-  # Default is the root folder.
-  folder: Formula
-
-  # Caveats for the user of your binary.
-  # Default is empty.
-  # caveats: "How to use this binary"
-
-  # Your app's homepage.
-  # Default is empty.
-  homepage: "https://github.com/Ibotta/sopstool"
-
-  # Your app's description.
-  # Default is empty.
-  description: "A multi-file wrapper for Mozilla sops"
-
-  # Setting this will prevent goreleaser to actually try to commit the updated
-  # formula - instead, the formula file will be stored on the dist folder only,
-  # leaving the responsibility of publishing it to the user.
-  # Default is false.
-  skip_upload: false
-
-  # Packages your package depends on.
-  dependencies:
-    - sops
-
-  # So you can `brew test` your formula.
-  # Default is empty.
-  test: |
-    system "#{bin}/sopstool version"
+nfpms:
```

Comment:
made a couple changes after inspecting the output of the tool per version deprecation notices